active/0000-pluggableLintTestGenerator.md

```diff
@@ -0,0 +1,61 @@
+- Start Date: 2015-05-19
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+Provide a unified API for linting for complete choice of testing framework and linting platform.
+
+# Motivation
+
+Allow user to have full control over what testing language and what linter they prefer.
+Currently ember-cli is tied into using JSHint always rather than being fully agnostic.
+
+# Detailed design
+
+In ember-cli addons main file.
+
+Expose a test generator API for the linting addon to call:
+```
+  lintTree: function(type, tree) {
```

Comment:
There is no reason why `this.testGenerator` can't be called more than once; that was how I was expecting the blocking aspect to be resolved.